src/components/login/Landing.vue

```diff
@@ -64,6 +65,12 @@ export default {
       }
     }
   }
+
+  .support-links {
+    margin-top: 30px;
+    text-align: center;
+    line-height: 1.6rem;
```

Comment:
1.6rem here is equivalent to: `line-height: toRem(23px);`
For consistency, we are using `toRem()` for the measurements.

Reply:
Fixed